vm-testing/inventory-sample

```diff
@@ -1,7 +0,0 @@
-[rhtas]
```

Comment:
What is the reason for removing this file in this PR?

Reply:
:anguished: mistake. Thank you for spotting this.